src/pages/en/core-concepts/routing.md

```diff
@@ -94,6 +94,30 @@ In this example, a request for `/withastro/astro/tree/main/docs/public/favicon.s
 }
 ```
 
+## Excluding pages
+
+You can exclude pages, or even whole directories from being built by prefixing their names with an underscore (`_`).
+
+This allows you to create private pages, and also to co-locate tests, utilities, and components with their related pages, preventing them from being built into html files and placed into the `dist/` directory.
+
+### Example: Excluding pages
```

Comment:
```suggestion
#### Example: Excluding pages
```

What do you think? If we add one more to the header, then it doesn't show up in the sidebar. I don't think having it show up as its own sub-heading in the sidebar adds anything.

Reply:
I agree, but then there's a semantics issue because we're skipping a heading level.
Maybe removing the heading altogether would be better? That would make it less consistent with the rest of the page, but because it's not a very large section it might not be noticeable.

Reply:
Sure, I'd be fine with removing it entirely!